cmd/utils/flags.go

```diff
@@ -187,7 +187,11 @@ var (
 	}
 	DeveloperPeriodFlag = cli.IntFlag{
 		Name:  "dev.period",
-		Usage: "Block period to use in developer mode (0 = mine only if transaction pending)",
+		Usage: "Block period for proof-of-authority network to use in developer mode (0 = mine only if transaction pending)",
+	}
+	DeveloperEthashFlag = cli.BoolFlag{
+		Name:  "dev.ethash",
```

Comment:
@meowsbits do you think `--dev.pow` makes more sense.

Also, I was thinking if anyone might want to test a flow having the ethash and not the fake one. In this case, we can allow this cli param to pass a value like `ethash`, `fakePoW`.

Reply:
Oh yea, interesting line of thought. Following this... What happens with `--miner.threads` values? With my first implementation, this value was co-opted to configure the emission rate.

I'm remembering now that `--fakepow` is an available flag... since that _can_/_could_ be used in conjunction with `--dev.[ethash|pow]`, maybe...

`--dev.[ethash|pow]` enables `--fakepow` by default, since not requiring actual CPU dedication seems usually desireable, making `--miner.threads` configure the emission rate in seconds. This could be overridden with `--fakepow=false`, where `--miner.threads` would revert to configuring the number of dedicated CPUs. 

In general, if we can pick sane defaults that a constrained `--dev.[ethash|pow]` would configure, and then allow adjacent/existing flags to handle further, overriding configuration, then I think we can avoid a rabbit hole of a plethora of development modes (which are really just a bundle of configuration options (flag and genesis)).

Reply:
Great ideas. So this will leave us with:

- `--dev.[ethash|pow]`
- `--fakepow`
- `--miner.threads`

Which then can be used like:

- `--dev.[ethash|pow] --miner.threads 1`, this will use ethash
- `--dev.[ethash|pow] --fakepow --miner.threads 1`, this will fakepow (we should document this nicely in the `--dev.[ethash|pow]`cli arg)

where `--miner.threads` if not set will default to:

1. follow the logic used on miner
2. set to **1** if above logic is complicated and coupled to a module

---
What do you think @meowsbits

Reply:
Have I misunderstood, or is `--dev.[ethash|pow]` supposed to be `--dev.[ethash|poa]`?

Reply:
By default you will use `--dev` which will enable PoA. If you prefer to have a PoW chain then you will use `--dev --dev.[ethash|pow]`.

Do you have other preferences/recommendations @BelfordZ?

Reply:
We (I?) were using `[ethash.pow]` to signal that the term was undecided (and that they're basically equivalent).  Leaning to `pow` now for Etchash vs. Ethash considerations.